src/main/scala/com/agoda/kafka/connector/jdbc/JdbcSourceConnector.scala

```diff
@@ -12,7 +12,7 @@ import scala.util.{Failure, Success, Try}
 
 class JdbcSourceConnector extends SourceConnector {
   private val logger = LoggerFactory.getLogger(classOf[JdbcSourceConnector])
-
+  Class.forName("org.postgresql.Driver")
```

Comment:
The driver should be loaded automatically, is it required to explicitly do this?